src/Spark.sol

```diff
@@ -13,10 +21,12 @@ contract Spark is ImpactEvaluator {
         _grantRole(DEFAULT_ADMIN_ROLE, 0x646ac6F1941CAb0ce3fE1368e9AD30364a9F51dA); // @bajtos
         _grantRole(DEFAULT_ADMIN_ROLE, 0xa0e36151B7074A4F2ec31b741C27E46FcbBE5379); // @patrickwoodhead
         _grantRole(DEFAULT_ADMIN_ROLE, 0x3ee4A552b1a6519A266AEFb0514633F289FF2A9F); // @juliangruber
-        // 30 minutes
-        setNextRoundLength(30 * 2);
-        // 200 FIL / month
-        setRoundReward(200 ether / (43800 / nextRoundLength * 2));
+
+        setNextRoundLength(roundLengthMinutes * (60 / blockTimeSeconds));
+
+        uint roundsInAMonth = minutesInAMonth / roundLengthMinutes;
```

Comment:
Should we move this to a top-level constant too?

Reply:
I wondered about this too. Since it's a computed value, I thought it could be part of the function. But either way works. What do you prefer?

Reply:
Will follow up in a separate PR

Reply:
Both have the same gas cost, and especially for constructors gas doesn't really matter anyway. I'll leave it as is then 👍